### PR TITLE
Flag examples in cosmology sub-package

### PR DESCRIPTION
**Pull request by lglattly**
### Description

This pull request is to flag examples in the (previously copy edited) cosmology sub-package for possible use in the example gallery.

### CODE REVIEW

File: docs/cosmology/index.rst [hunk 1]
```diff
@@ -86,6 +97,13 @@ special case). However, you cannot work with this class directly, as
 you must specify a dark energy model by using one of its subclasses
 instead, such as `~astropy.cosmology.FlatLambdaCDM`.
 
+Examples
+--------
+
+..
+  EXAMPLE START
+  Working with the FLRW Class Through Subclass FlatLambdaCDM
```
Comment (adrn) [inline docs/cosmology/index.rst]:
```suggestion
  Working with the FlatLambdaCDM Class
```

File: docs/cosmology/index.rst [hunk 2]
```diff
@@ -132,10 +150,18 @@ They also accept arrays of redshifts:
...
-`~astropy.cosmology.LambdaCDM`. A variety of standard
-cosmologies with the parameters already defined are also available
+methods and attributes available.
+
+..
+  EXAMPLE END
+
+..
+  EXAMPLE START
+  Working with the FLRW Class Through Subclass LambdaCDM
```
Comment (adrn) [inline docs/cosmology/index.rst]:
```suggestion
  Working with Non-flat Universes with the LambdaCDM Class
```

File: docs/cosmology/index.rst [hunk 3]
```diff
@@ -199,7 +228,13 @@ instance are immutable — that is, if you want to change, say,
 this more convenient, a ``clone`` operation is provided, which
 allows you to make a copy with specified values changed.
 Note that you cannot change the type of cosmology with this operation
-(e.g., flat to non-flat). For example:
+(e.g., flat to non-flat).
+
+..
+  EXAMPLE START
+  Making New Cosmology Instances with the Clone Operation
```
Comment (adrn) [inline docs/cosmology/index.rst]:
```suggestion
  Making New Cosmology Instances with the .clone() Method
```

File: docs/cosmology/index.rst [hunk 4]
```diff
@@ -210,11 +245,23 @@ Note that you cannot change the type of cosmology with this operation
...
 If you know a cosmological quantity and you want to know the
-redshift which it corresponds to, you can use ``z_at_value``:
+redshift which it corresponds to, you can use ``z_at_value``.
+
+Example
+^^^^^^^
+
+..
+  EXAMPLE START
+  Finding the Redshift at a Given Value of a Cosmological Quantity
```
Comment (adrn) [inline docs/cosmology/index.rst]:
```suggestion
  Compute the Redshift at a Given Universe Age
```

File: docs/cosmology/index.rst [hunk 5]
```diff
@@ -331,6 +381,13 @@ Therefore, if you need to compute many distances in such a cosmology and
 performance is critical, it is particularly useful to calculate them on
 a grid and use interpolation.
 
+Examples
+^^^^^^^^
+
+..
+  EXAMPLE START
+  Calculating the Contribution of Photons and Neutrinos with Cosmology
```
Comment (adrn) [inline docs/cosmology/index.rst]:
```suggestion
  Calculating the Contribution of Photons and Neutrinos to the Energy Density
```

**Comment by lglattly** [comment]
Thanks for the suggestions!

**Pull request merged**